lib/config/options/index.ts

```diff
@@ -822,7 +822,13 @@ const options: RenovateOptions[] = [
       type: 'string',
       format: 'uri',
     },
-    supportedManagers: ['helm-requirements', 'helmv3', 'helmfile', 'gitlabci'],
+    supportedManagers: [
```

Comment:
Could `docker-compose`, `gitlabci` and `kubernetes` managers also be added here? (if not just all managers under the docker bullet point from the docs)

Reply:
they don't support registry aliases yet, needs seperate PR'S

Reply:
@viceice Then this pr won't solve my issue, could you mark it as such?

Reply:
We are currently implementing all the other managers.